src/analytics/types.ts

```diff
@@ -58,6 +58,13 @@ export enum Action {
   CreateDiscountClick = "CREATE_DISCOUNT_CLICK",
   CreateQuicklinkClick = "CREATE_QUICKLINK_CLICK",
   ApplyFinanceClick = "APPLY_FINANCE_CLICK",
+
```

Comment:
i think these will come under onchange or smth so should probably add that in events

Reply:
these events will be logged when the data inside them is valid. That's why kept them like this.

Reply:
I think Fatima is saying that when you log these actions, it will come under the onClick event from the `Events` enum, which is incorrect.

you need to add Event in `Events` something like: `isValid` or `onChange` or something relevant and consume that instead and then also change your `Action` values to something like: `ADD_NEW_EMAIL_INPUT`, `ADD_NEW_PHONE_INPUT`, or something that would make more sense